src/main/java/org/minions/devfund/bruno/html/HtmlReader.java

```diff
@@ -0,0 +1,29 @@
+package org.minions.devfund.bruno.html;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Queue;
+
+/*
+ * SD2x Homework #2
+ * This class contains a method that will read an HTML file and convert it to a Queue of HtmlTags.
+ * It is simply provided as a convenience class for you to use during your testing.
+ * You do not need to submit this code.
+ */
+
+public class HtmlReader {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)